vendor-extra/LiberoPageBundle/src/ViewConverter/ItemListVisitor.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Libero\LiberoPageBundle\ViewConverter;
+
+use FluentDOM\DOM\Element;
+use FluentDOM\DOM\Node;
+use FluentDOM\DOM\Node\NonDocumentTypeChildNode;
+use Libero\ViewsBundle\Views\SimplifiedVisitor;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverter;
+use Libero\ViewsBundle\Views\ViewConverterVisitor;
+
+final class ItemListVisitor implements ViewConverterVisitor
+{
+    use SimplifiedVisitor;
+
+    private $converter;
+
+    public function __construct(ViewConverter $converter)
+    {
+        $this->converter = $converter;
+    }
+
+    protected function doVisit(Element $object, View $view) : View
+    {
+        return $view->withArgument(
+            'nodes',
+            array_map(
+                function (Node $child) use ($view) : View {
+                    return $this->converter->convert(
+                        $child,
+                        '@LiberoPatterns/paragraph.html.twig',
+                        $view->getContext()
+                    );
+                },
+                iterator_to_array($object->getElementsByTagNameNS('http://libero.pub', 'item-ref'))
+            )
+        );
+    }
+
+    protected function expectedTemplate() : string
+    {
+        return '@LiberoPatterns/text.html.twig';
```

Comment:
`text` is becoming too generic here, it rather is `generic-nodes.html.twig` or similar?

Reply:
Indeed. Only used here due to the lack of a normal pattern, but I have wondered about making it a macro instead.